justfile

```diff
@@ -1,9 +1,10 @@
 # Install the environment to develop the extension.
 prelude:
-        pip3 install pyo3-pack
-        cargo install pyo3-pack
-        pip3 install virtualenv
         virtualenv -p $(which python3) .env
+        source .env/bin/activate
+
+        pip3 install pyo3-pack pytest pytest-benchmark
+        pip3 install virtualenv
```

Comment:
This must be moved before line 3 (`virtualenv -p $(which python3) .env`, right?

Reply:
Good catch